test/blackbox-tests/test-cases/pkg/lock-out-of-sync.t

```diff
@@ -41,6 +41,7 @@ We add the bar dependency to the test package
 
 It fails as we have not regenerated the lock:
   $ dune build
+  File "dune.lock", line 1, characters 0-0:
```

Comment:
Isn't it more confusing to have a pointer to the file with specific line and characters instead of just the instruction to run `dune pkg lock`? I would understand with just the filename (as a dir) but here, it might create the inverted effect and push the user to have a look at the directory. Wouldn't it be possible to have `Directory "dune.lock"` instead of `File "dune.lock", line 1, characters 0-0`?

Reply:
I could pick the `lock.dune` file inside `dune.lock`.

Reply:
You are right. It might be more understandable from a user perspective.